canonical-web-and-design/vanilla-framework
# Added empty state - 1st variant

## Done

-  Variant 1 empty state is added.
- Empty state is added to docs.

Fixes #1010

## QA

- Open empty state [example](https://vanilla-framework-4274.demos.haus/docs/examples/layouts/empty-state/no-content)
- Verify its implemented according to [design](https://discourse.ubuntu.com/t/variant-a-no-content-empty-state/24619)
- Open [docs](https://vanilla-framework-4274.demos.haus/docs/base/empty-state)
- Verify empty state's no content variant is documented correctly.
- Open [whats new](https://vanilla-framework-4274.demos.haus/docs/whats-new) section.
- Verify "empty state" is added.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
